conda_smithy.recipe/meta.yaml

```diff
@@ -35,6 +35,8 @@ requirements:
     - scrypt
     - license-expression
     - libarchive
+  run_constrained:
+    - mamba 0.2.*
```

Comment:
You'll have to edit the actual feedstock by hand FYI. This does not get copied over in the release process.